h2o-py/h2o/debug.py

```diff
@@ -0,0 +1,40 @@
+from .tree import H2OTree
+
+
+def _equals_nan_friendly(l1, l2):
+    for u, v in zip(l1, l2):
+        if u is not v and u != v:
+            return False
+    return True
+
+
+def equal_gbm_model_tree_structure(gbm_1, gbm_2):
+    """
+    Check if gbm models has the same tree structure. The purpose of this method is to debug GBM reproducibility.
+
+    :param gbm_1: First model to check
+    :param gbm_2: Second model to check
+    :return: True if tree structure is equal
+    """
+    if gbm_1.ntrees != gbm_2.ntrees:
```

Comment:
Can you add check that the models are gbm models?

Reply:
Yes